bricks/ev3rt/README.md

```diff
@@ -121,3 +122,16 @@ make ev3rt -j && cp bricks/ev3rt/build/uImage /media/$USER/EV3RT/uImage && umoun
 
 For now, there is just the REPL and several builtin MicroPython modules.
 Sensors and motors are not yet enabled.
+
+## About TOPPERS/EV3RT and TOPPERS/HRP3 (general information not related to PyBricks)
```

Comment:
```suggestion
## About TOPPERS/EV3RT and TOPPERS/HRP3 (general information not related to Pybricks)
```

'b' should not be capitalized in Pybricks.

Reply:
Thanks for the quick review. I have fixed the original commit and force-pushed.